js_modules/dagit/packages/core/src/runs/useComputeLogFileKeyForSelection.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import {useQueryPersistedState} from '../hooks/useQueryPersistedState';
+
+import {
+  IRunMetadataDict,
+  matchingComputeLogKeyFromStepKey,
+  ILogCaptureInfo,
+} from './RunMetadataProvider';
+
+export function useComputeLogFileKeyForSelection({
```

Comment:
Just moved this logic out to a hook so that it didn't need to be replicated to show the logs in a second place